.gitignore

```diff
@@ -45,3 +45,10 @@ coverage
 
 # Visual Studio Code
 .vscode
+
+# devops files
+
+bin/terrafile
+
+# Downloaded terraform modules
+terraform/aks/vendor/
```

Comment:
Missing final new line (as on other files). If using VScode, consider using [these options](https://dfedigital.atlassian.net/wiki/spaces/BaT/pages/1971126288/Team) to do it automatically.